web-ui/src/components/admin/roles/Roles.jsx

```diff
@@ -62,10 +62,30 @@ const Roles = () => {
   memberProfiles?.sort((a, b) => a.name.localeCompare(b.name));
 
   useEffect(() => {
-    if (roles) {
```

Comment:
I don't think we want to get roles from state now. I think we only want to get them from a query parameter.